_posts/2017-05-01-five-lessons-learned.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Five Lessons Learned During My First Year as a Technical Director"
+date:   2017-05-01 12:00:00 -0400
+categories: culture management leadership
+author: Patrick Hereford
+---
+
+Making the leap from an engineer to a director can be daunting. There are so
+many other responsibilities that get placed on you as a director that it can be
+overwhelming. The hardest shift, I feel, is the switch from spending most of your
+time and energy coding to solving other types of problems. Most of
+that time shifts towards higher priority responsibilities like budgeting,
+planning, product strategy, mentoring, recruitment and so much more.
+Even though coding is still enjoyable for those making this switch, it can be
+rather far down the list of largest impact a director can provide to their
+organization. But like engineering, one should have the mindset of finding
+challenges and continually find learning opportunities within their new role.
```

Comment:
`continually` embracing(?) challenges and ~~continually find~~ learning opportunities